qiskit/wrapper/jupyter/jupyter_magics.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=import-error
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""A module of magic functions"""
+
+import sys
+import time
+import threading
+from IPython.display import display
+from IPython.core import magic_arguments
+from IPython.core.magic import cell_magic, Magics, magics_class
+import ipywidgets as widgets
+import qiskit
+
+
+@magics_class
+class StatusMagic(Magics):
+    """A class of status magic functions.
+    """
+    @cell_magic
+    @magic_arguments.magic_arguments()
+    @magic_arguments.argument(
+        '-i',
+        '--interval',
+        type=float,
+        default=2,
+        help='Interval for status check.'
+    )
+    def qiskit_job_status(self, line='', cell=None):  # pylint: disable=W0613
+        """A Jupyter magic function to check the status of a Qiskit job instance.
+        """
+        args = magic_arguments.parse_argstring(self.qiskit_job_status, line)
+        # Split cell lines to get LHS variables
+        _cell_lines = cell.split('\n')
```

Comment:
Can you change the variable names in this function so they do not start with an underscore? Variables declared inside a function have the scope of that function already, and treating them as private by prepending the `_` is not needed.